plugins/modules/win_gpo_link.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021 Sebastian Gruber ,dacoso GmbH All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_gpo_link
+author: Sebastian Gruber (@sgruber94)
```

Comment:
Needs to have `version_added: 1.4.0` to signal when this module was added to `community.windows`.